lib/scsi_defs.h

```diff
@@ -3,8 +3,12 @@
 
 #include <cstdint>
 
+#include "absl/base/attributes.h"
+
 namespace scsi_defs {
 
+  typedef uint64_t LunAddress;
```

Comment:
is this used somewhere

Reply:
It's not currently used as LunAddresses are reported as variable sized data. The intention was for the typedef to be used later when variable sized fields are handled.

Reply:
ah I see. thanks for clarifying!